src/kw_include.sh

```diff
@@ -23,6 +23,8 @@ function include()
   local filepath="$1"
   local fullpath
   local varname
+  local hashname
+  local hash
```

Comment:
`hash` is a shell builtin, I recommend you change the name of this variable. Maybe `hash_str`?